firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -1844,63 +1844,120 @@ class BrowserViewController: UIViewController,
         }
     }
 
-    private func creditCardAutofillSetup(_ tab: Tab, didCreateWebView webView: WKWebView) {
+    /// Returns whether the credit card autofill feature flag is enabled from Nimbus
+    func autofillCreditCardNimbusFeatureFlag() -> Bool {
+        return featureFlags.isFeatureEnabled(.creditCardAutofillStatus, checking: .buildOnly)
+    }
+
+    /// Returns whether credit card autofill settings are enabled based on user defaults key KeyAutofillCreditCardStatus
```

Comment:
I think this comment (and some of the other comments in this PR) could possibly be removed because the function names below them are named well and are easy to understand. 

I was going to bring this up on our Thursday meeting re: team philosophy on code comments, since I've noticed industry shifting to preferring descriptive function names and modularization code into smaller bits of logic  contained in their own functions to give more descriptive function names and help with the flow of reading code without comments.

Comments still serve an important purpose when documenting frameworks or APIs, or commenting on bits of code where what's happening isn't easy to understand with naming alone, but whenever we add a comment along with a line of code there's a risk of that comment not getting updated when the logic changes in the future.

Anyway, happy to hop on a call if you want to clarify anything tomorrow! It might also give me a good idea of what to cover on this topic for Thursday :)

Reply:
@dataports agreed, if the function name is well versed then I don't see the point of adding comments unless we do the docc part where comments help with knowing the documentation 

please add it as an agenda item and it would be good to discuss

Reply:
I removed my beautiful comments :)